it/test/controllers/IndexControllerISpec.scala

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2025 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import org.scalatestplus.play.PlaySpec
+import play.api.http.Status._
+import play.api.libs.ws.{DefaultWSCookie, WSClient}
+import play.api.mvc.{Session, SessionCookieBaker}
+import play.api.test.Helpers.{await, defaultAwaitTimeout}
+import utils.ISpecBase
+
+class IndexControllerISpec extends PlaySpec with ISpecBase {
+
+  lazy val wsClient: WSClient = app.injector.instanceOf[WSClient]
+  val session                 = Session(Map("authToken" -> "abc123", "role" -> "admin"))
```

Comment:
Hey @C-Thomp  we would require only authToken in session.. we can remove role.. sorry just added for testing

Reply:
good spot ill sort that :)